docs/scala_proto_library.md

```diff
@@ -32,25 +32,59 @@ generated by the [ScalaPB compiler](https://github.com/scalapb/ScalaPB).
 
 To configure ScalaPB options, configure a different `scala_proto_toolchain` and declare it in a `BUILD` file:
 
-```python
+```starlark
 load("@io_bazel_rules_scala//scala_proto:scala_proto_toolchain.bzl", "scala_proto_toolchain")
-load("@io_bazel_rules_scala//scala:providers.bzl", "declare_deps_provider")
 
 scala_proto_toolchain(
-    name = "scala_proto_toolchain_configuration",
+    name = "scala_proto_toolchain",
     with_grpc = False,
     with_flat_package = False,
     with_single_line_to_string = False,
     visibility = ["//visibility:public"],
-    dep_providers = [":my_grpc_deps", ":my_compile_deps"],
 )
 
 toolchain(
     name = "scalapb_toolchain",
-    toolchain = ":scala_proto_toolchain_configuration",
+    toolchain = ":scala_proto_toolchain",
     toolchain_type = "@io_bazel_rules_scala//scala_proto:toolchain_type",
     visibility = ["//visibility:public"],
 )
+```
+
+
+### `scala_proto_toolchain` Toolchain Attributes
+
+| Attribute name                | Description                                           |
+| ----------------------------- | ----------------------------------------------------- |
+| name                          | `Name, required` <br> A unique name for this toolchain.
+| with_grpc                     | `boolean, optional (default False)` <br> Enables generation of grpc service bindings for services.
+| with_flat_package             | `boolean, optional (default False)` <br> When true, ScalaPB will not append the protofile base name to the package name.
+| with_single_line_to_string    | `boolean, optional (default False)` <br> Enables generation of toString() methods that use a single line format.
+| blacklisted_protos            | `List of labels, optional` <br> List of protobuf targets to exclude from recursive building.
+| code_generator                | `Label, optional (has default)` <br> Which code generator to use. A sensible default is provided.
+| named_generators              | `String dict, optional` <br>
+| extra_generator_dependencies  | `List of labels, optional` <br>
```

Comment:
Maybe add explanation that this contains a list of jars where named generators are loaded from.